<!DOCTYPE html>
<!-- saved from url=(0072)https://physiology.med.cornell.edu/faculty/khelashvili/lab/research.html -->
<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
		<meta name="viewport" content="initial-scale=1"><meta name="viewport" content="user-scalable=yes,width=1080"><title>George Khelashvili, Ph.D.</title>
		
		<meta name="viewport" content="width=device-width, initial-scale=1">
		<!--[if lte IE 8]><script src="assets/js/ie/html5shiv.js"></script><![endif]-->
		<link rel="stylesheet" href="./Lab_files/main.css">
		<!--[if lte IE 9]><link rel="stylesheet" href="assets/css/ie9.css" /><![endif]-->
	</head>
	<body>
		<div id="page-wrapper">
			<div id="header-wrapper">
				<div class="container">
					<div class="row">
						<div class="12u">
							<header id="header">
								<h1><a href="https://physiology.med.cornell.edu/faculty/khelashvili/lab/research.html#" id="logo">Khelashvili Research Group</a></h1>
								<nav id="nav">
									<a href="https://physiology.med.cornell.edu/faculty/khelashvili/lab/index.html">Home</a>
									<a href="https://physiology.med.cornell.edu/faculty/khelashvili/lab/research.html" class="current-page-item">Research</a>
									<a href="https://physiology.med.cornell.edu/faculty/khelashvili/lab/publications.html">Publications</a>
									<a href="https://physiology.med.cornell.edu/faculty/khelashvili/lab/people.html">People</a>
									<a href="https://physiology.med.cornell.edu/faculty/khelashvili/lab/resources.html">Resources</a>
									<a href="https://physiology.med.cornell.edu/faculty/khelashvili/lab/contact.html">Contact</a>
								</nav>
							</header>
						</div>
					</div>
				</div>
			</div>
			<div id="main">
				<div class="container">
					<div class="row main-row">
						<div class="8u 12u(mobile)">
							
							<section class="left-content">
								<h2>Research topics</h2>
							</section>

							<h3>Mechanisms of lipid scrambling by GPCR proteins</h3>
						  <p></p>
							<section>
								<ul class="image-list">
									<li>
										<a href="https://physiology.med.cornell.edu/faculty/khelashvili/lab/research.html#"><img src="./Lab_files/scrambling.png" alt="" width="400" class="left"></a>
										<p>Several Class-A G protein-coupled receptor (GPCR) proteins act as constitutive phospholipid scramblases catalyzing the transbilayer translocation of &gt;10,000 phospholipids per second when reconstituted into synthetic vesicles. To address the molecular mechanism by which these proteins facilitate rapid lipid scrambling, we carry out large-scale ensemble atomistic molecular dynamics simulations of GPCRs embedded in membranes of various lipid composition. On the example of opsin, prototypical class-A GPCR, we discovered that in the process of scrambling, lipid headgroups traverse a dynamically revealed hydrophilic pathway in the region between transmembrane helices 6 and 7 of the protein while their hydrophobic tails remain in the bilayer environment. This is reminiscent of a credit card (the magnetic strip of the card corresponds to the lipid headgroup, while the body of the card represents lipid acyl chains) swiping through a credit card reader (protein). We  build quantitative kinetic models of the translocation process based on Markov State Model analysis and find that key residues on the lipid translocation pathway are conserved within the Class-A GPCR family. These results illuminate new aspects of GPCR structure and dynamics while providing a rigorous basis for the design of variants of these proteins with defined scramblase activity.</p>
									</li>
								</ul>
								<h1>Publications:</h1>
								<p>Morra G, Razavi AM, Pandey K, Weinstein H, Menon AK, Khelashvili G. <strong>Mechanisms of Lipid Scrambling by the G Protein-Coupled Receptor Opsin.</strong>  Structure 2018, 26:1-12 <a href="https://www.ncbi.nlm.nih.gov/pubmed/29290486">PMID 29290486</a></p>
							</section>

							<h3>Deficient oligomerization of rhodopsin mutants associated with retinitis pigmentosa disease</h3>
						  <p></p>
							<section>
								<ul class="image-list">
									<li>
										<a href="https://physiology.med.cornell.edu/faculty/khelashvili/lab/research.html#"><img src="./Lab_files/RHM-figure-new.png" alt="" width="400" class="left"></a>
										<p>The visual pigment rhodopsin is both a G protein-coupled receptor (GPCR) and a critical structural component of the outer segment of photoreceptor cells. Mutations in rhodopsin are the most common cause of autosomal dominant retinitis pigmentosa (RP), a blinding disease that afflicts more than 1.5 million individuals world-wide. Rhodopsin can activate the visual signaling cascade as a monomer, but it self-associates to form dimers and higher order multimers that have been proposed to be important for phototransduction. Experimental studies discovered that certain rhodopsin mutants that cause RP appear to function normally as visual pigments but unlike wild type rhodopsin, they fail to dimerize when reconstituted into lipid vesicles. These results lead to a hypothesis that the lack of dimerization in these mutants can prevent normal photoreceptor function, leading eventually to loss of photoreceptor cells, retinal degeneration and RP pathology. The goal of computational research in the lab is to identify the molecular mechanism of rhodopsin dimerization and to elucidate why it fails in certain RP-associated rhodopsin mutants. The computations involve extensive probing of membrane-mediated protein-protein interactions and protein aggregation for the wild type and the RP-mutant rhodopsin constructs by means of coarse-grained molecular dynamics simulations combined with continuum level theoretical formulation of membrane elasticity and hydrophobic mismatch.</p>
									</li>
								</ul>
								<h1>Publications:</h1>
								<p> Mondal S, Johnston J, Wang H, Khelashvili G, Filizola M, Weinstein H. <strong>Membrane Driven Spatial Organization of GPCRs. </strong> Nature Scientific Reports 2013 3:2909 <a href="https://www.ncbi.nlm.nih.gov/pubmed/24105260">PMID 24105260</a></p>
							</section>

							
						<h3>Allosteric mechanisms of functional transitions in neurotransmitter transporters</h3>
						  <p></p>
							<section>
								<ul class="image-list">
									<li>
										<a href="https://physiology.med.cornell.edu/faculty/khelashvili/lab/research.html#"><img src="./Lab_files/DAT-figure_for_review.png" alt="" width="300" class="left"></a>
										<p>Members of Neurotransmitter:Sodium Symporter (NSS) family of proteins, such as dopamine (DAT), serotonin (SERT), and norepinephrine (NET) transporters are responsible for the clearance of neurotransmitters from the synaptic cleft, and for their translocation back into the presynaptic nerve terminal. The mechanism of neurotransmitter uptake is made possible by coupling the transmembrane Na+ and Cl- gradients to the uphill transport of the respective substrate. The physiological reuptake function of NSS proteins have been shown to be complemented by their involvement in the reverse transport of substrate (efflux). The efflux is triggered by binding of psychostimulants, such as amphetamine (AMPH), and the association of AMPH with DAT is considered to set in motion a complex molecular machinery that creates favorable local conditions for recruitment of kinases appropriate for the phosphorylation of the DAT N-terminus leading to dopmaine efflux. One of the major components of this regulatory machinery include highly charged phosphatidylinositol 4,5-bisphosphate (PIP2) lipids. Our overall goal is to understand in molecular detail the alosteric mechanisms of uptake and efflux processes and how these are regulated protein-membrane interactions. We use large-scale moleculad dynamics simulations and Markov State Modeling to build quantitative kinetic models of functional transitions in NSS transporters, such as DAT. This allows us to identify how the functional terminal segments of the DAT are allosterically involved in the release process and can differentially regulate bi-directional substrate transport.  </p>
									</li>
								</ul>
								<h1>Publications:</h1>
								<p> Razavi AM, Khelashvili G, Weinstein H. <strong>A Markov State-based Quantitative Kinetic Model of Sodium Release from the Dopamine Transporter. </strong> Nature Scientific Reports 2017, 7:40076 <a href="https://www.ncbi.nlm.nih.gov/pubmed/28059145">PMID 28059145</a></p>
								<p> Khelashvili G, Stanley N, Sahai MA, Medina J, LeVine MV, Shi L, De Fabritiis G, Weinstein H. <strong>Spontaneous Inward Opening of the Dopamine Transporter Is Triggered by PIP2‐Regulated Dynamics of the N‐Terminus. </strong> ACS Chemical Neuroscience 2015, 6:1825−1837 <a href="https://www.ncbi.nlm.nih.gov/pubmed/26255829">PMID 26255829</a></p>
							</section>
		

							<h3>Quantification of membrane mechanical properties from molecular dynamics simulations</h3>
						  <p></p>
							<section>
								<ul class="image-list">
									<li>
										<a href="https://physiology.med.cornell.edu/faculty/khelashvili/lab/research.html#"><img src="./Lab_files/TOC_v5.png" alt="" width="300" class="left"></a>
										<p>Elastic properties of membranes play major roles in many cellular processes that require membrane reshaping, including exocytosis and endocytosis. They also regulate the organization and function of many membrane-associated proteins. The determination of these properties is therefore key for our understanding of many physiological processes at the molecular level. We have developed a novel computational method, termed RSF (Real-Space Fluctuations), to quantify bending rigidity and lipid tilt modulus, the two commonly sought elastic properties of lipid assemblies. To calculate these moduli, we analyze the fluctuations in the tilt and splay degrees of freedom as sampled in the molecular dynamics trajectories in real space. The method has a number of important advantages over existing alternative computational techniques that extract bending and tilt moduli from MD simulations: i) it is not limited to single component membranes but rather is applicable to lipid mixtures; ii) it can be used to characterize lipid systems in different thermodynamic phases (i.e. liquid ordered or disorder states) and of arbitrary geometries (i.e. hexagonal phase); and iii) the local nature of the formulation allows calculations of the material properties for compositionally asymmetric and/or phase-separated lipid membrane as well as for specific membrane domains, as encountered for example in the vicinity of a protein insertion or adsorption site. Currently the work is ongoing to enhance the methodology to include calculations of other imporant components of the membrane deformation energy such as bilayer compressibility modulus. </p>
									</li>
								</ul>
								<h1>Publications:</h1>
								<p> Doktorova M, Harries D, Khelashvili G. <strong>Determination of bending rigidity and tilt modulus of lipid membranes from real-space fluctuation analysis of molecular dynamics simulations. </strong> Physical Chemistry Chemical Physics 2017, 19:16806-16818 <a href="https://www.ncbi.nlm.nih.gov/pubmed/28627570">PMID 28627570</a></p>
								<p> Johner N, Harries D, Khelashvili G. <strong>Implementation of a methodology for determining elastic properties of lipid assemblies from molecular dynamics simulations. </strong> BMC Bioinformatics 2016, 17:161 <a href="https://www.ncbi.nlm.nih.gov/pubmed/27071656">PMID 27071656</a></p>
							</section>
							
							
							<h3>Effect of environment on organization and function of transmembrane proteins</h3>
						  <p></p>
							<section>
								<ul class="image-list">
									<li>
										<a href="https://physiology.med.cornell.edu/faculty/khelashvili/lab/research.html#"><img src="./Lab_files/TOC_graphics_NEW.png" alt="" width="400" class="left"></a>
										<p>Structure/function relationships in many transmembrane proteins are often studied in artificial environments, such as detergent micelles or non-lamellar lipidic assemblies, i.e. lipid cubic phases. But how these non-native conditions influence the experimental measurements is not well understood. To evaluate the relation between the properties of the proteins measured in such arteficial media and in native conditions we study with various computational approaches structural and dynamic characteristics of different classes of transmembrane proteins (i.e. transporters, GPCRs) in detergent micelles and in lipid cubic phases.  
										<a href="https://physiology.med.cornell.edu/faculty/khelashvili/lab/research.html#"><img src="./Lab_files/TOC_graphics_cubic phase.png" alt="" width="200" class="left"></a>
											</p><p>On the example of LeuT, we showed how functional properties of membrane proteins can be influenced by the identity and concentration of detergents used in the solubilization process, and demontrated a crucial role of annular lipids, that are presumably retained during the transfer of the protein from the native lipid bilayer to the detergent environment, in the experimentally measured functional phenotypes. We also showed how concentration and type of monoolein lipids can influence large scale organization of GPCR proteins in lipidic cubic phases in which these proteins are crystallized for structure determination. </p>
									</li>
								</ul>
								<h1>Publications:</h1>
								<p> LeVine MV, Khelashvili G, Shi L, Quick M, Javitch JA, Weinstein H. <strong>Role of Annular Lipids in the Functional Properties of Leucine Transporter LeuT Proteomicelles. </strong> Biochemistry 2016, 55:850-859 <a href="https://www.ncbi.nlm.nih.gov/pubmed/26811944">PMID 26811944</a></p>
								<p> Khelashvili G, LeVine M, Shi L, Quick M, Javitch J, Weinstein H. <strong>The Membrane Protein LeuT in Micellar Systems: Aggregation Dynamics and Detergent Binding to the S2 Site. </strong> Journal of the American Chemical Society 2013 135(38):14266–14275 <a href="https://www.ncbi.nlm.nih.gov/pubmed/23980525">PMID 23980525</a></p>
								<p> Johner N, Mondal S, Morra G, Caﬀrey M, Weinstein H, Khelashvil G. <strong>Protein and Lipid Interactions Driving Molecular Mechanisms of in meso Crystallization. </strong> Journal of the American Chemical Society 2014 136(8):3271-84 <a href="https://www.ncbi.nlm.nih.gov/pubmed/24494670">PMID 24494670</a></p>
							</section>
							
							
							
						</div>
						<div class="4u 12u(mobile)">

							<section>
								<ul class="small-image-list">
									<li>
										<a href="https://physiology.med.cornell.edu/faculty/khelashvili/lab/research.html#"><img src="./Lab_files/george_thumb.gif" alt="" class="left"></a>
										<h4>George Khelashvili, Ph.D.</h4>
										<p>Assistant Professor at the Department of Physiology and Biophysics, Assistant Professor in Computational Biophysics at Weill Cornell Medicine in New York.</p>
									</li>
								</ul>
							</section>

							<section>
								<div>
									<div class="row">
										<div class="6u 12u(mobile)">
											<ul class="link-list">
												<li><a href="http://vivo.med.cornell.edu/display/cwid-gek2009">Research Profile</a></li>
												<li><a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Khelashvili%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25739722">PubMed profile</a></li>
											</ul>
										</div>
										<div class="6u 12u(mobile)">
											<ul class="link-list">
												<li><a href="https://physiology.med.cornell.edu/people/george-khelashvili-ph-d/">Faculty page</a></li>
												<li><a href="https://physiology.med.cornell.edu/">Departmental website</a></li>
											</ul>
										</div>
									</div>
								</div>
							</section>

							<section>
								<a href="https://physiology.med.cornell.edu/faculty/khelashvili/lab/contact.html" class="button">Contact</a>
							</section>

						</div>
					</div>
				</div>
			</div>
			<div id="footer-wrapper">
				<div class="container">
					<div class="row">
						<div class="12u">

							<div id="copyright">
								© George Khelashvili. All rights reserved. | Design: <a href="http://html5up.net/">HTML5 UP</a>
							</div>

						</div>
					</div>
				</div>
			</div>
		</div>

		<!-- Scripts -->
			<script src="./Lab_files/jquery.min.js"></script>
			<script src="./Lab_files/skel.min.js"></script>
			<script src="./Lab_files/skel-viewport.min.js"></script>
			<script src="./Lab_files/util.js"></script>
			<!--[if lte IE 8]><script src="assets/js/ie/respond.min.js"></script><![endif]-->
			<script src="./Lab_files/main.js"></script>

	
<div id="titleBar"><a href="https://physiology.med.cornell.edu/faculty/khelashvili/lab/research.html#navPanel" class="toggle"></a><span class="title">Khelashvili Research Group</span></div><div id="navPanel"><nav><a class="link depth-0" href="https://physiology.med.cornell.edu/faculty/khelashvili/lab/index.html" style="-webkit-tap-highlight-color: rgba(0, 0, 0, 0);"><span class="indent-0"></span>Home</a><a class="link depth-0" href="https://physiology.med.cornell.edu/faculty/khelashvili/lab/research.html" style="-webkit-tap-highlight-color: rgba(0, 0, 0, 0);"><span class="indent-0"></span>Research</a><a class="link depth-0" href="https://physiology.med.cornell.edu/faculty/khelashvili/lab/publications.html" style="-webkit-tap-highlight-color: rgba(0, 0, 0, 0);"><span class="indent-0"></span>Publications</a><a class="link depth-0" href="https://physiology.med.cornell.edu/faculty/khelashvili/lab/people.html" style="-webkit-tap-highlight-color: rgba(0, 0, 0, 0);"><span class="indent-0"></span>People</a><a class="link depth-0" href="https://physiology.med.cornell.edu/faculty/khelashvili/lab/resources.html" style="-webkit-tap-highlight-color: rgba(0, 0, 0, 0);"><span class="indent-0"></span>Resources</a><a class="link depth-0" href="https://physiology.med.cornell.edu/faculty/khelashvili/lab/contact.html" style="-webkit-tap-highlight-color: rgba(0, 0, 0, 0);"><span class="indent-0"></span>Contact</a></nav></div></body></html>